src/test/java/core/basesyntax/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ReaderServiceImplTest {
+    private static final String INPUT_FILE_PATH = "src/main/resources/input.csv";
+    private static final String EMPTY_FILE_PATH = "src/main/resources/empty.csv";
+    private ReaderServiceImpl readerService;
+    private List<String> records;
+
+    @BeforeEach
+    public void setUp() {
+        readerService = new ReaderServiceImpl();
+        records = new ArrayList<>();
+        records.add("type,fruit,quantity");
+        records.add("b,banana,20");
+        records.add("b,apple,100");
+        records.add("s,banana,100");
+        records.add("p,banana,13");
+        records.add("r,apple,10");
+        records.add("p,apple,20");
+        records.add("p,banana,5");
+        records.add("s,banana,50");
+    }
+
+    @Test
+    public void read_rightFile_Ok() {
```

Comment:
```suggestion
    public void read_rightFile_ok() {
```
user lowercase after underscore everywhere